Definitions.md

```diff
@@ -0,0 +1,16 @@
+#Definition Of Terms
+
+##Job
```

Comment:
In case of nested workflows, is the running instance of an inner "nested workflow task" a TaskInstance or WorkflowInstance?

It would be great to have a superclass term for TaskInstance and WorkflowInstance. Can we perhaps use "Job" for this, rather than an alias for TaskInstance?